phantom_low_p:
  - 40
  - 0
  - 117

phantom_high_p:
  - 180
  - 88
  - 255

inserter_low_p:
  - 0
  - 0
  - 0

inserter_high_p:
  - 132
  - 255
  - 67

tentacle_low_p:
  - 0
  - 0
  - 0
  
tentacle_high_p:
  - 132
  - 255
  - 67